README.md

```diff
@@ -191,9 +194,11 @@ end
 And this cleaned value is available in observations in the final published result:
 
 ```ruby
-class MyExperiment < ActiveRecord::Base
+class MyExperiment
   include Scientist::Experiment
 
+  # ...
```

Comment:
@natesholland it is the folded part of the experiment boiler plate (a representation thereof)

```
  attr_accessor :name

  def initialize(name: name)
    @name = name
  end
```